superset-frontend/src/dashboard/components/SliceHeaderControls/SliceHeaderControls.test.tsx

```diff
@@ -170,6 +173,16 @@ test('Should "export to CSV"', async () => {
   expect(props.exportCSV).toBeCalledWith(371);
 });
 
+test('Should "export to XLSX"', async () => {
+  const props = createProps();
+  renderWrapper(props);
+  expect(props.exportXLSX).toBeCalledTimes(0);
+  userEvent.hover(screen.getByText('Download'));
+  userEvent.click(await screen.findByText('Export to .XLSX'));
```

Comment:
@vitoldi pls, fix the unit test. Use `Excel` instead of `.XLSX`
```
Error: Unable to find an element with the text: Export to .XLSX.
This could be because the text is broken up by multiple elements.
In this case, you can provide a function for your text matcher to make your matcher more flexible.
```

Reply:
![1](https://github.com/apache/superset/assets/82170648/83cf2e57-2a06-4e5f-a012-5754472e7f18)